lib/types.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+
+export type Case = 'AVERAGE' | 'BEST_CASE' | 'MEDIAN' | 'WORST_CASE'
+
+export type Grid = {
+  contains: Function,
+  data: Int32Array,
+  north: number,
+  west: number,
+  height: number,
+  width: number,
+  zoom: number
+}
+
+export type ImageData = {
+  data: Uint8ClampedArray,
+  height: number,
+  width: number
+}
+
+export type Journey = {
+  journey_id: string | number,
+  journey_name: string | number,
+  segments: Array<any>
+}
+
+export type Origin = {
+  data: Int32Array,
+  index: Int32Array,
+  nMinutes: number,
+  nStops: number,
+  radius: number,
+  x: number,
```

Comment:
I don't believe so as that would be a runtime constraint on the value contained in a JavaScript Number. Google `flowtype integers` and let me know if you see anything promising. Remember I'm only a few days ahead on `flow`y things 😉